readme.md

```diff
@@ -18,34 +18,66 @@
+If you only want to download the Blender and STL files `DO NOT USE download ZIP` GitHub button, since it is not compatible with LFS (Large File Storage), but instead get the files from the [latest release](https://github.com/inputlabs/alpakka_case/releases/latest) package.
 
 To use Git with this project it is required to install Git [Large File Storage](https://git-lfs.github.com).
 
 
-## Repeated and mirrored parts
-Some parts require to print 2x or 4x units, and some of them are mirrored. For simplicity, and to avoid mistakes, we export the STL files with these repetitions and mirroring baked in.
+## Parts hierarchy
 
-If you want to split these to arrange them better in the printer bed:
+### Main assembly
+- `alpakka.blend` - Alpakka controller assembly.
 
-**In PrusaSlicer**
-- Right click in object.
-- Split > To objects.
+### Case
+- `case_front.blend` - Front case + cutouts.
+- `case_back.blend` - Back case + cutouts.
+- `case_cover.blend` - Rear bay cover.
+- `anchor.blend` - Anchors holding the cases together.
+- `lock.blend` - Screws holding the cases together.
 
-**In Cura**
-- Open Marketplace.
-- Install `Mesh Tools` plugin.
-- Right click in object.
-- Mesh Tools > Split model into parts.
+### Buttons
+- `button_dpad.blend` - Dpad buttons.
+- `button_abxy.blend` - ABXY buttons.
+- `button_select.blend` - Select/start buttons.
+- `button_home.blend` - Home button.
 
-## File labels
+### Triggers
+- `trigger_R1.blend` - L1 and R1 shoulder triggers.
+- `trigger_R2.blend` - L2 and R2 triggers.
+- `trigger_R4.blend` - L4 and R4 triggers.
+
+### Control widgets
+- `hexagon.blend` - Touch sensitive surface.
+- `thumbstick.blend` - Thumbstick cap.
+- `dhat.blend` - 8-directional switch hat.
+- `scrollwheel.py (build123d)` - Scrollwheel, core, and holder.
+
+### Additional
+- `soldering_stand.blend` - Tool to hold the PCB while soldering.
+- `pcb.blend` - Reference model of the PCB.
+- `shared.blend` - Common geometry nodes / modifiers shared by all parts.
+
+
+## Exported filename labels
 - `primary`: Primary color or material according to Alpakka reference design color pattern.
 - `secondary`: Primary color or material according to Alpakka reference design color pattern.
 - `any`: Non visible part or tool that can be printed in any material, even recycled ;)
 - `015mm`: 0.15mm layer height, default for most prints.
 - `007mm`: 0.07mm layer height, for parts that require extra finesse.
 - `020mm`: 0.20mm layer height, for tools that we want to print fast.
-- `(brim)`: Consider using a brim for improved bed adhesion.
-- `(support)` Requires support material, check the manual!.
+- `2x`, `4x`: To be printed multiple times.
 
 Except for the colors (which is up to you how to combine), it is very recommended to follow these indications, and to check the [Manual](https://inputlabs.io/devices/alpakka/manual/diy_case) for more details.
+
+
+## Migration to Build123d
```

Comment:
Altogether it is not really clear what it means to the reader. For now this is acceptable for this PR; but an update to the website and the corresponding section (https://inputlabs.io/alpakka/manual/diy_case) should be done a.s.a.p.